src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/publicApi/ToModelTransformers.kt

```diff
@@ -144,6 +143,14 @@ private fun transform(entity: AdditionalConditionPss): ModelPublicAdditionalCond
     text = entity.text,
     category = entity.category,
     categoryShort = entity.categoryShort,
-    requiresInput = entity.requiresInput,
+    requiresUserInput = entity.requiresInput,
   )
 }
+
+fun String.getPolicyVersion() =
+  when {
+    this == "1.0" -> PolicyVersion.V1_0
+    this == "2.0" -> PolicyVersion.V2_0
+    this == "2.1" -> PolicyVersion.V2_1
+    else -> error("No matching policy found")
+  }
```

Comment:
mmmh... rather than have to map all these here (and have to remember to update here when adding a new policy)

We should use the version va l:  `PolicyVersion.values().find {  ...blah   } ?: throw blah` 

also worth adding context to the error `error("some message that can refer to $this to explain what was not found")`

Reply:
to take the example from electronic monitoring type:

```kt
  companion object {
    fun find(value: String): ElectronicMonitoringType? = entries.find { it.value == value }
  }
```

Reply:
Done